File: program/convert-hours-to-minutes/convert-hours-to-minutes.py
def convertHoursToMinutes(hours: float) -> float:
    """
    convert hours to minutes
    Arguments:
        hours:float
    Returns:
        minutes:float
    """

    return hours * 60


if __name__ == "__main__":
    hours = float(input("Type number of hours: "))
    print("Number of minutes is:", convertHoursToMinutes(hours))


# contributed by MedMly20208 Mohamed Moulay

File: program/convert-hours-to-minutes/ConvertHoursToMinutes.cs
using System;

/**
 * ConvertHoursToMinutes
 */
public class ConvertHoursToMinutes {

    static void Main(string[] args)
    {
        Console.WriteLine("Enter Hours ");
        int input = Convert.ToInt32(Console.ReadLine());
        Console.WriteLine("Input: " + input);
        Console.WriteLine("Output: " + (input * 60));
    }
}
